store.json

```diff
@@ -5,43 +5,67 @@
     "image": "alexellis/inception",
     "name": "inception",
     "fprocess": "python3 index.py",
-    "network": "func_functions"
+    "network": "func_functions",
+    "repo_url": ""
```

Comment:
is it not this? https://github.com/faas-and-furious/inception-function

Reply:
Thanks!